.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,40 @@
+{
+	"name": "RedwoodJS & Postgres (Community)",
+
+	// Update the 'dockerComposeFile' list if you have more compose files or use different names.
+	"dockerComposeFile": "docker-compose.yml",
+
+	// The 'service' property is the name of the service for the container that VS Code should
+	// use. Update this value and .devcontainer/docker-compose.yml to the real service name.
+	"service": "app",
+
+	// The optional 'workspaceFolder' property is the path VS Code should open by default when
+	// connected. This is typically a volume mount in .devcontainer/docker-compose.yml
+	"workspaceFolder": "/workspace",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Add the IDs of extensions you want installed when the container is created.
+			"extensions": [
+				"redwoodjs.redwood",
```

Comment:
The extension isn't maintained and we've stopped suggesting it in the create-redwood-app template:
```suggestion
```

Reply:
Thanks for letting me know!